### DISCUSSION TITLE
Desktop install

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#12524] [General]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

16

### Storage for /

1tb

### Storage for /nsm

2tb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have installed so desktop. It does not launch the GUI however. It shows since this is not a manager the pill balues to enable security onion desktop must be set manually.  Desktop:GUI: enabled:true. 
How do i enable this?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Have you ran a highstate on the Desktop after install? `sudo so-checkin`
